algebird-spark/src/main/scala/com/twitter/algebird/spark/package.scala

```diff
@@ -10,16 +10,22 @@ import scala.reflect.ClassTag
  */
 package object spark {
 
+import org.apache.spark.sql.Dataset
+
   /**
    * spark exposes an Aggregator type, so this is here to avoid shadowing
    */
   type AlgebirdAggregator[A, B, C] = Aggregator[A, B, C]
   val AlgebirdAggregator = Aggregator
 
-  implicit class ToAlgebird[T](val rdd: RDD[T]) extends AnyVal {
+  implicit class ToAlgebirdRDD[T](val rdd: RDD[T]) extends AnyVal {
```

Comment:
can we not change the name of this? this will break source and binary compatibility for people.